security/certificate_types_descriptions/service-ca-certificates.adoc

```diff
@@ -27,6 +27,11 @@ The service CA expiration of 26 months is longer than the expected upgrade inter
 A manually-rotated service CA does not maintain trust with the previous service CA. You might experience a temporary service disruption until the pods in the cluster are restarted, which ensures that pods are using service serving certificates issued by the new service CA.
 ====
 
+[IMPORTANT]
+====
+Applications that use a different trust store, such as JavaKeyStore, need to be capable of accepting new CA certifications dynamically into it's trust store, or else when automated rotation occurs, pods will require a restart.
```

Comment:
> Applications that use a different trust store

The Jira says "Develop a java app which has service ca-certificate dependency on a separate keystore managed by the client". The issue only exists when both conditions: 1. the store is different/separate; 2. the different/separate store has dependency on service-ca ca certificate. So, how about preciser statement `Applications that use a different ...store which has dependency on service-ca certificate`?

Reply:
Fixed. Thanks!